static/src/stylesheets/module/facia/_items.scss

```diff
@@ -49,6 +52,10 @@
     }
 }
 .fc-slice__item--no-mpu {
+    -webkit-box-flex: 0 !important;
+    -moz-box-flex: 0 !important;
+    -webkit-flex: 0 !important;
+    -ms-flex: 0 !important;
```

Comment:
yeah, but doesn't allow adding `!important`. Did add it as a arg to that helper, but it was getting more complicated than I thought necessary